src/pages/Explore/index.tsx

```diff
@@ -37,52 +41,81 @@ const TitleContainer = styled.div`
   margin-right: auto;
   display: flex;
 `
+const NavWrapper = styled.div<{ isInfoExplorePageEnabled: boolean }>`
+  display: flex;
+  max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
+  margin: 0 auto;
+  margin-bottom: ${({ isInfoExplorePageEnabled }) => (isInfoExplorePageEnabled ? '16px' : '20px')};
+  color: ${({ theme }) => theme.neutral3};
+  flex-direction: row;
 
-const NavItem = styled(ThemedText.MediumHeader)<{ active?: boolean }>`
+  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  ${({ isInfoExplorePageEnabled }) =>
+    isInfoExplorePageEnabled &&
+    css`
+      /* Use large media breakpoint so expanded search bar doesn't push tabs to next line */
+      @media screen and (max-width: ${LARGE_MEDIA_BREAKPOINT}) {
+        flex-direction: column;
+        gap: 16px;
+      }
+    `};
+`
+const TabBar = styled(AutoRow)`
+  gap: 24px;
+  @media screen and (max-width: ${({ theme }) => theme.breakpoint.md}px) {
+    gap: 16px;
+  }
+`
+const TabItem = styled(ThemedText.HeadlineMedium)<{ active?: boolean }>`
   align-items: center;
   color: ${({ theme, active }) => (active ? theme.neutral1 : theme.neutral2)};
   cursor: pointer;
-  display: flex;
-  justify-content: space-between;
   transition: ${({ theme }) => `${theme.transition.duration.medium} ${theme.transition.timing.ease} color`};
 `
-const FiltersContainer = styled.div<{ isExplore: boolean }>`
+const FiltersContainer = styled.div<{ isInfoExplorePageEnabled: boolean }>`
   display: flex;
   gap: 8px;
   height: 40px;
 
   @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
-    ${({ isExplore }) => !isExplore && 'order: 2;'}
-    ${({ isExplore }) => isExplore && 'justify-content: space-between;'}
+    ${({ isInfoExplorePageEnabled }) => !isInfoExplorePageEnabled && 'order: 2;'}
   }
-`
-const DropdownFilterContainer = styled(FiltersContainer)`
-  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
-    justify-content: flex-start;
+
+  @media screen and (max-width: ${({ theme }) => theme.breakpoint.md}px) {
+    ${({ isInfoExplorePageEnabled }) => isInfoExplorePageEnabled && 'justify-content: space-between;'}
   }
 `
-const SearchContainer = styled(FiltersContainer)<{ isExplore: boolean }>`
-  ${({ isExplore }) => !isExplore && 'margin-left: 8px;'}
+const DropdownFilterContainer = styled(FiltersContainer)<{ isInfoExplorePageEnabled: boolean }>`
+  ${({ isInfoExplorePageEnabled }) =>
+    isInfoExplorePageEnabled
+      ? css`
+          @media screen and (max-width: ${({ theme }) => theme.breakpoint.md}px) {
+            justify-content: flex-start;
+          }
+        `
+      : css`
+          @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
```

Comment:
whats the px difference between `theme.breakpoint.md` and `MEDIUM_MEDIA_BREAKPOINT` it would be modifying the existing experience but it would be ideal to deprecate the instances of these Details/Explore page only breakpoints

Reply:
`theme.breakpoint.md` is 768, `MEDIUM_MEDIA_BREAKPOINT` is 720. 

If we're ok with changing the existing experience, can deprecate these breakpoints in a diff PR. Am talking with design about this!